fineract-provider/src/main/java/org/apache/fineract/accounting/glaccount/data/GLAccountData.java

```diff
@@ -132,14 +135,6 @@ private GLAccountData(String name, Long parentId, String glCode, Boolean manualE
         this.allowedExpensesTagOptions = null;
```

Comment:
These explicit null assignments make no sense... just to say: that's empty code and no reason why not to replace these constructors... also having multiple constructors is an anti-pattern, so let's try again with the Lombok annotation and get rid of this...